client/package.json

```diff
@@ -64,7 +64,7 @@
     "eslint-plugin-react-hooks": "^4.2.0",
     "sass": "^1.52.1",
     "serve": "^12.0.1",
-    "typescript": "^4.1.2",
+    "typescript": "^4.7.2",
```

Comment:
Needed to update in order to include .getDisplayMedia() in the typescript definitions